### DISCUSSION TITLE
Copilot always saying "Encountering temporary issues"

### DISCUSSION THREAD

**Discussion by AyanShekhar** [community/community#9912] [Copilot Conversations]
Copilot always saying "Encountering temporary issues"

**Comment by thethiny** [comment]
It's a glitch in the UI, not a glitch that prevents it from working. If you type something then reject the suggestions it'll say encountering issues even if there are no issues. Or if you erase after typing, or are in the middle of the line.

**Reply by davecheney** [reply]
This should be corrected in the next release of the extension.

**Comment by ISMAILBOUADDI** [comment]
I just used copilot till now with python, i faced this problem (encountering issues) many times, in first time i thought it's just network problem

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by mlongoria1** [reply]
Here is what I believe are the relevant parts of the output. I have also highlighted where the error is coming from and where the pulldown is with red rectangles. So if any future viewers are confused about where this info is coming from(as I was,) now they will know.

![PG4yyl9SOo](https://user-images.githubusercontent.com/38320817/162000025-2edd88b0-dd6d-47f7-9548-479340ffa350.png)


It seems to me that there is some sort of pricing issue still left behind from the OpenAI portion of the source code that still needs to be removed when they created Codex. Unless M$ is planning to roll out a pricing structure for future use I think this should be considered a bug and removed from Codex.

**Reply by manuthebyte** [reply]
I'm also running into this error atm. 
Referring to #14370
